pkgs/development/libraries/gobject-introspection/wrappers/g-ir-compiler.sh

```diff
@@ -0,0 +1,9 @@
+#! @bash@
+# shellcheck shell=bash
+
+# try to check if we're in a nix build
+if [ -n "${name:-}" ]; then
```

Comment:
What is this about? I don't feel good about this.

Reply:
Other wrappers also don't work very well outside a Nix build, granted, but such special casing I don't think is the right way to fix that.